examples/CustomVocabularies.java

```diff
@@ -19,18 +20,24 @@ public static void main(String[] args) {
 
     // Create a custom vocabulary for your submission
     CustomVocabulary customVocabulary =
-        new CustomVocabulary(Arrays.asList("Robert Berwick", "Noam Chomsky", "Evelina Fedorenko"));
+            new CustomVocabulary(Arrays.asList("Robert Berwick", "Noam Chomsky", "Evelina Fedorenko"));
+
+    // Set up the notification url if desired
+    String notificationUrl = "https://example.com";
+    // Authorization header is optional; use it if needed to access the callback notification url
+    Map<String, String> notificationAuth = new HashMap<>();
+    notificationAuth.put("Authorization", "Bearer <callback_token>");
 
     CustomVocabularySubmission customVocabularySubmission = new CustomVocabularySubmission();
     customVocabularySubmission.setCustomVocabularies(Arrays.asList(customVocabulary));
-    customVocabularySubmission.setCallbackUrl("https://example.com");
+    customVocabularySubmission.setNotificationConfig(notificationUrl, notificationAuth);
     customVocabularySubmission.setMetadata("My first submission");
 
     CustomVocabularyInformation submittedCustomVocabularyInfo;
 
     try {
       submittedCustomVocabularyInfo =
-          customVocabulariesClient.submitCustomVocabularies(customVocabularySubmission);
+              customVocabulariesClient.submitCustomVocabularies(customVocabularySubmission);
```

Comment:
Is this too far indented? Lots of stuff in this file appears to have been indented father than it was

Reply:
switched the continuation indent back from 8 to 4